aviary/mission/gasp_based/ode/flight_path_ode.py

```diff
@@ -161,6 +167,27 @@ def setup(self):
                                        promotes_outputs=subsystem.mission_outputs(**kwargs))
 
         if analysis_scheme is AnalysisScheme.SHOOTING:
+            # temporary until a tagged release of OM contains the necessary fix
+            from aviary.utils.test_utils.check_om_version import CheckForOMSubmodelFix
+            self.submodel_fix = CheckForOMSubmodelFix()
+            if not self.submodel_fix:
+                self.add_subsystem('mass_trigger',
+                                   om.ExecComp(
+                                       'mass_trigger = OEM + payload + reserve_fuel + descent_fuel',
+                                       mass_trigger={'val': 0, 'units': 'lbm'},
+                                       OEM={'val': 0, 'units': 'lbm'},
+                                       payload={'val': 0, 'units': 'lbm'},
+                                       reserve_fuel={'val': 0, 'units': 'lbm'},
+                                       descent_fuel={'val': 0, 'units': 'lbm'},
+                                   ),
+                                   promotes_inputs=[
+                                       ('OEM', Aircraft.Design.OPERATING_MASS),
+                                       ('payload', Aircraft.CrewPayload.PASSENGER_PAYLOAD_MASS),
+                                       ('reserve_fuel', Mission.Design.RESERVE_FUEL),
+                                       'descent_fuel',
+                                   ],
```

Comment:
Part 1 out of 2 of this particular comment:
Could you add `mass_trigger` in the `promotes_outputs` here? Or is that not possible?